src/core/Card/Card.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import useStyles from './styles';
+
+interface CardProps {
+  glow: boolean;
+  width: string;
+  height: string;
+}
+
+const Card: React.FC<CardProps> = ({ glow, width, height, children }) => {
```

Comment:
can you change the name so that it doesn't conflict with Mui Card component

Reply:
Resolved